spec/support/repository_factory.rb

```diff
@@ -0,0 +1,91 @@
+# frozen_string_literal: true
+
+require_relative "vcr"
+
+class StubRepository
+  attr_reader :full_name, :branches
+
+  def repo_file(repo_name, path)
```

Comment:
Could we rename this to be just `#file`? Since it's on the `StubRepository` already.